tailwind.config.js

```diff
@@ -47,7 +48,11 @@ module.exports = {
           6: '#171717',
         },
 
-      }
+      },
+      fontFamily:{
+        'sans': ['Inter', ...defaultTheme.fontFamily.sans],
+        //'sans': ['Montserrat', ...defaultTheme.fontFamily.sans],
```

Comment:
again, make sure to leave out or uncomment code comments

Reply:
done